Library/Homebrew/cmd/update-report.rb

```diff
@@ -214,16 +216,18 @@ def report
         new_tap = tap.tap_migrations[name]
         @report[status.to_sym] << full_name unless new_tap
       when "M"
-        begin
-          formula = Formulary.factory(tap.path/src)
-          new_version = formula.pkg_version
-          old_version = FormulaVersions.new(formula).formula_at_revision(@initial_revision, &:pkg_version)
-          next if new_version == old_version
-        rescue FormulaUnavailableError
-          # Don't care if the formula isn't available right now.
-          nil
-        rescue Exception => e # rubocop:disable Lint/RescueException
-          onoe "#{e.message}\n#{e.backtrace.join "\n"}" if ARGV.homebrew_developer?
+        unless ENV["HOMEBREW_DISABLE_LOAD_FORMULA"]
```

Comment:
Thoughts on doing an early return here instead?

Reply:
We could, yeah. I suppose skipping printing the new/changed/deleted formula list entirely doesn't really matter.

Reply:
Aye 👍